### ISSUE TITLE
Attribute order

### ISSUE THREAD

**Issue by TobiasSico** [wojtpl2/ExtendedXmlSerializer#5]
Hey Wojtpl2,

your Project is really nice and useful, but what I need to do is to define the order of the Attributes in the XML. You wrote that the ExtendedXmlSerializer 1.4.0 support "XmlElement", so I try [XmlElement(Order = 1)], but it doesn't work for me. 
Is there any other way to define the order of the Attributes?
Now the order of the Attributes is the order how the Properties are defined in the C# class. 

Examples:
how it works now:
C# class
class Test{
  string B {get; set;}
  string A {get; set;}
}
XML Output:
<?xml version="1.0" encoding="utf-8"?>
<Test>
  <B></B>
  <A></A>
</Test>

What I would like to do:
C# class
class Test{
  [XmlElement(Order = 2)]
  string B {get; set;}
  [XmlElement(Order = 1)]  
  string A {get; set;}
}
XML Output:
<?xml version="1.0" encoding="utf-8"?>
<Test>
  <A></A>
  <B></B>
</Test>

Thank you in advance for your answer!

**Comment by wojtpl2** [comment]
Thank. Ok, I can support Order parameter too. I will add this function in version 1.4.1. I'll try make it today or tomorrow.

**Comment by TobiasSico** [comment]
Thank you. 
I forgot something to ask you. In my Case one Property is in the Base-class and the other Property is in the derived class. 
Is this also possible?

Example what I would like to do:
class BaseTest{
  [XmlElement(Order = 1)]
  string B;
}
class Test : BaseTest{
  [XmlElement(Order = 2)]
  string A
}

**Comment by wojtpl2** [comment]
I published version 1.4.1. This version support Order parameter.
You can set Order on property in Base Class and Sub class too. But in your example you don't need any attribute becouse new version of ExtendedXmlSerializer write first all property from base class then all property from sub class. If you want you can reverse order, like this:

```
class BaseTest{
[XmlElement(Order = 2)]
string B;
}
class Test : BaseTest{
[XmlElement(Order = 1)]
string A
}
```
